.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Set default behaviour, in case users don't have core.autocrlf set.
+* text=auto
+
+*.sh text eol=lf
```

Comment:
Have you tried if this works on windows?

Reply:
I didn't actually, but I went with what @nvoxland has for the liquibase core repo (https://github.com/liquibase/liquibase/blob/master/.gitattributes), so I assumed it should work exactly the same.